scw/errors.go

```diff
@@ -360,6 +362,22 @@ func (e *ResourceNotFoundError) GetRawBody() json.RawMessage {
 	return e.RawBody
 }
 
+type ResourceLockedError struct {
+	Resource   string `json:"resource"`
+	ResourceID string `json:"resource_id"`
+
+	RawBody json.RawMessage `json:"-"`
+}
+
+// IsScwSdkError implements the SdkError interface
+func (e *ResourceLockedError) IsScwSdkError() {}
```

Comment:
I don't understand this line 

why doesn't it do anything ?

Reply:
It implements an interface